### PR TITLE
feat: document schema registry (r5.0)

### PR DESCRIPTION
**Pull request by thalesmg**
# Note: the zh_CN translation needs updating after en_US version is stabilized.

### CODE REVIEW

File: en_US/data-integration/schema-registry-example-protobuf.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+5. In the **Topic** field, type `protobuf_out` as the destination topic.
+6. In the **Payload** field, type message content template: `${protobuf_person}`.
+
+This action sends the Protobuf encoded user message to the topic `protobuf_out`. `${protobuf_person}` is a variable placeholder that will be replaced at runtime with the value of the result of `schema_encode` (a binary value).
+
+### Prepare Device-Side Code
+
+Once the rules have been created, you can simulate the data for testing.
+
+The following code uses the Python language to fill a user message, encode it as binary data, then send it to the `protobuf_in` topic. See [full code](https://gist.github.com/thalesmg/c5f03f99f982401d16ef6583e30144fa) for details.
```
Comment (Meggielqk) [inline en_US/data-integration/schema-registry-example-protobuf.md:147]: I just have a doubt here. In the encoding scenario, the message still needs to be encoded as binary data in this step? Is that correct? @thalesmg

File: en_US/data-integration/schema-registry-example-protobuf.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+   ```shell
+   $ pip3 install protobuf paho-mqtt
+   
+   $ python3 protobuf_mqtt_sub.py
+   Connected with result code 0
+   protobuf_out name: "Shawn"
+   id: 1
+   email: "shawn@example.com"
+   ```
```
Comment (Meggielqk) [inline en_US/data-integration/schema-registry-example-protobuf.md:185]: In this code example, there is no binary message like the one in the Avro example? Just be curious. @thalesmg

Comment (thalesmg) [reply en_US/data-integration/schema-registry-example-protobuf.md:185]: It is encoded as binary, just the example code was not printing the raw binary payload.  I've updated the example code and docs to include the binary printout.

File: en_US/data-integration/schema-registry-example-protobuf.md (hunk 1 above)
Comment (thalesmg) [reply en_US/data-integration/schema-registry-example-protobuf.md:147]: Yes, it is encoded as binary here.  I've added a print statement to print it out.

File: en_US/data-integration/schema-registry-example-protobuf.md (hunk 2 above)
Comment (Meggielqk) [reply en_US/data-integration/schema-registry-example-protobuf.md:185]: Ok, thank you!

File: en_US/data-integration/schema-registry-example-avro.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+2. Create an Avro schema using the following parameters:
+
+   - **Name**: `avro_user`
+
+   - **Type**: `Avro`
+
+   - **Schema**:
+
+     ```protobuf
```
Comment (thalesmg) [inline en_US/data-integration/schema-registry-example-avro.md]:
```suggestion
     ```json
```

File: en_US/data-integration/schema-registry-example-avro.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+### Check Rule Execution Results
+
+1) In the Dashboard, select **Diagnose** -> **WebSocket Client**. 
+2) Fill in the connection information for the current EMQX instance.
+   - If you run EMQX locally, you can use the default value.
+   - If you have changed EMQX's default configuration. For example, the configuration change on authentication can require you to type in a username and password.
+
+3. Click **Connect** to connect to the EMQX instance as an MQTT client.
+
+4. In the **Publish** area, type `avro_out` in the **Topic** field and type the following message in the **Payload** field:
```
Comment (thalesmg) [inline en_US/data-integration/schema-registry-example-avro.md]:
```suggestion
4. In the **Publish** area, type `avro_in` in the **Topic** field and type the following message in the **Payload** field: 
```

File: en_US/data-integration/schema-registry-example-protobuf.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+### Check Rule Execution Results
+
+1) In the Dashboard, select **Diagnose** -> **WebSocket Client**.
+2) Fill in the connection information for the current EMQX instance.
+   - If you run EMQX locally, you can use the default value.
+   - If you have changed EMQX's default configuration. For example, the configuration change on authentication can require you to type in a username and password.
+
+3. Click **Connect** to connect to the EMQX instance as an MQTT client.
+
+4. In the **Publish** area, type `protobuf_out` in the **Topic** field and type the following message in the **Payload** field:
```
Comment (thalesmg) [inline en_US/data-integration/schema-registry-example-protobuf.md]:
```suggestion
4. In the **Publish** area, type `protobuf_in` in the **Topic** field and type the following message in the **Payload** field:
```

File: zh_CN/data-integration/schema-registry-example-avro.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+1. 在 Dashboard 页面，点击左侧导航目录中的 **问题分析** -> **WebSocket 客户端**。
+
+2. 填写当前 EMQX 的连接信息。
+
+   - 如果 EMQX 在本地运行，可直接使用默认配置。
+   - 如果您修改过 EMQX 的默认配置，如修改过访问规则的配置，则需要输入用户名和密码。
+
+3. 点击**连接**，作为 MQTT 客户端连接到 EMQX。
+
+4. 在**发布**区域，在**主题**中输入 `avro_out`，在 **Payload** 中输入以下消息：
```
Comment (thalesmg) [inline zh_CN/data-integration/schema-registry-example-avro.md]:
```suggestion
4. 在**发布**区域，在**主题**中输入 `avro_in`，在 **Payload** 中输入以下消息：
```

File: zh_CN/data-integration/schema-registry-example-protobuf.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+1. 在 Dashboard 页面，点击左侧导航目录中的**问题分析** -> **WebSocket 客户端**。
+
+2. 填写当前 EMQX 的连接信息。
+
+   - 如果 EMQX 在本地运行，可直接使用默认配置。
+   - 如果您修改过 EMQX 的默认配置，如修改过访问规则的配置，则需要输入用户名和密码。
+
+3. 点击**连接**，作为 MQTT 客户端连接到 EMQX。
+
+4. 在**发布**区域，在**主题**中输入 `protobuf_out`，在 **Paylod** 中输入以下消息：
```
Comment (thalesmg) [inline zh_CN/data-integration/schema-registry-example-protobuf.md]:
```suggestion
4. 在**发布**区域，在**主题**中输入 `protobuf_in`，在 **Paylod** 中输入以下消息：
```

File: dir.yaml [hunk 8]
```diff
@@ -213,11 +213,14 @@
...
     #   children:
     #     - data-integration/rule-configs
-    # - title_en: Schema Registry
-    #   title_cn: Schema 注册
-    #   edition: ee
-    #   children:
-    #     - data-integration/schema-registry
+
+- title_en: Schema Registry
+  title_cn: Schema 注册
```
Comment (wivwiv) [inline dir.yaml]:
```suggestion
  title_cn: Schema Registry
```

File: en_US/data-integration/assets/schema_registry/schema_registry1.svg
Comment (wivwiv) [inline en_US/data-integration/assets/schema_registry/schema_registry1.svg:1]: @Meggielqk Architecture diagram using emqx style

**Review by wivwiv — DISMISSED** [review]

**Review by wivwiv — APPROVED** [review]

**Pull request merged**